CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 <!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
 <!-- Please also include if this is a Bug Fix, Enhancement, or Feature -->
 
+# v1.39.1
+
+## Bug Fixes
+* Upgrades `go-retryablehttp` dependency to `v0.7.5` which address several bugs, see [changelog](https://github.com/hashicorp/go-retryablehttp/blob/v0.7.5/CHANGELOG.md).
```

Comment:
I would hoist this message to here since it fixes a perceived bug in go-tfe by @sebasslash [#813](https://github.com/hashicorp/go-tfe/pull/813)

```suggestion
* fixes an issue where the request body is not preserved during certain retry scenarios
```